### ISSUE TITLE
Keyboard suggestions does not work

### ISSUE THREAD

**Issue by Neizvestnyj** [kivy/kivy#7405]
**Software Versions**
* Python: 3.8
* OS: android
* Kivy: 2.0.0
* p4a: develop
* Kivy installation method: pip

The standard row that displays keyboard predictions does not appear on the keyboard.  It does not work with either the default keyboard or _Microsoft Swift_, the bug does not depend on the **api** version.  Below are screenshots showing which line with the word prediction I mean and how it looks in _kivy_

```python
from kivy.lang import Builder
from kivy.app import App

KV = '''
BoxLayout:
    padding: "10dp"
    TextInput:
        keyboard_suggestions: True
'''


class Test(App):
    def __init__(self, **kwargs):
        super().__init__(**kwargs)
        self.screen = Builder.load_string(KV)

    def build(self):
        return self.screen

Test().run()
```
![148967ED-D735-45DF-8A5B-EB4934F738C6](https://user-images.githubusercontent.com/40869738/110183954-08b48180-7e21-11eb-8073-bc6a9fdf47e3.jpeg)

![93BDF6B1-C4C1-4126-9DE3-4DACDE8D02EA](https://user-images.githubusercontent.com/40869738/110183958-09e5ae80-7e21-11eb-8979-2c394dc6ef02.jpeg)

https://user-images.githubusercontent.com/40869738/110184043-3dc0d400-7e21-11eb-8db9-bdf141f5f6df.MP4

**Comment by matham** [comment]
It should work with master. But also see https://github.com/kivy/kivy/issues/7337 for additional issues.

**Comment by Neizvestnyj** [comment]
@matham But in the master thread it does not work correctly https://github.com/kivy/kivy/issues/7415

**Comment by matham** [comment]
Ok, but this specific issue is fixed in master. You can try back-porting the fix to 2.0.0 if you can't use master because of the linked issue.

**Comment by Neizvestnyj** [comment]
> Any progress here?

No
